Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3
+
+RUN pip install pipenv
+
+
```

Comment:
Delete this whitespace (empty line).
You don't need 2 empty lines, one is enough, unless there is Docker convention I don't know of that tells to keep 2 empty lines in a case like this.
If there is such a convention, add a comment here and point me to it (paste a link or something).